package.json

```diff
@@ -29,7 +29,10 @@
     "@redhat-cloud-services/frontend-components": "^3.3.3",
     "@redhat-cloud-services/frontend-components-notifications": "^3.2.2",
     "@redhat-cloud-services/frontend-components-utilities": "^3.2.4",
+    "@typescript-eslint/eslint-plugin": "^5.3.0",
     "classnames": "^2.3.1",
+    "eslint-plugin-import": "^2.25.2",
+    "eslint-plugin-react-hooks": "^4.2.0",
```

Comment:
Might want to move these to the devDeps section.

Reply:
Thanks, fixed. :+1: